docs/2/technical/data-format/microshare-standards.md

```diff
@@ -224,31 +218,32 @@ Example :
 
 #### A. General Structure
 
-A large part of this data is simply derived from the settings of our device cluster, the others represent the data produced by the sensor.
+The majority of the information within the packet of data details the settings of the device cluster. The remaining pieces of information contain the data collected by the sensors. 
+
+This is how the device cluster is set up:
 
-Here is the set-up of this device cluster   : 
 
 {% include image.html url="\assets\img\dc-data-formatnew.png" width="800" description="dc" %}
 
 <br>
 
 #### B. {Data}
 
-Now let's talk about the `{Data}` part, which is the one with the most data.
+Now let's talk about the `{Data}`, which has the majority of the information.
 
-In this one are all the different parts: 
+The various components include: 
   - C. [Ipso](./#c-ipso)
   - D. [Meta/IoT](./#d-metaiot)
   - E. [Origin](./#e-origin)
   - F. [Unpacker](./#f-unpacker)
 
-But above all it includes simple data such as in this example `temperature`, `humidity`, `co2` ...
+`{Data}` includes the data from your devices such as the `temperature`, `humidity`, `co2` and so on. 
 
 All these values are explained with their units in the following table: 
 
 ##### > [Standards Data Table](./#3-standards-data-table)
 
-Information about the device itself that provided the data is sometimes given, including the device id and the battery of this sensor if it sends it in the payload. For example: 
+`{Data}` contains information about the sensor that provided the data payload. This usually includes the sensors device id, battery type, battery type, etc. The code supplied will look like: 
```

Comment:
about the device that provided the sensor data payload.
Note:
There is the common mistake of using 'sensor' and 'device' to be synonyms. They are not. A sensor is a thing that measures some physical property of the environment (it is usually one or more chips). Those sensors are combined with things like memory, radios & cpus to create a device. A device will have 1 or more sensors attached.